File: api-net/Models/SmsLog.cs
namespace LiftOtonom.Api.Models;

public class SmsLog
{
    public long Id { get; set; }
    public long TenantId { get; set; }
    public string Recipient { get; set; } = "";
    public string Message { get; set; } = "";
    public string? Status { get; set; }
    public string? ProviderId { get; set; }
    public string? TriggerType { get; set; }
    public int Cost { get; set; } = 1;
    public DateTime SentAt { get; set; }
}

File: api/app/Models/SmsLog.php
<?php
// app/Models/SmsLog.php

namespace App\Models;

use App\Models\Concerns\HasTenant;
use Illuminate\Database\Eloquent\Model;

class SmsLog extends Model
{
    use HasTenant;

    public $timestamps = false;

    protected $fillable = [
        'tenant_id', 'recipient', 'message', 'status',
        'provider_id', 'trigger_type', 'cost', 'sent_at',
    ];

    protected function casts(): array
    {
        return ['sent_at' => 'datetime', 'cost' => 'integer'];
    }
}
